documentation/modules/rn-2.6.0.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+Migrate virtual machines (VMs) from VMware vSphere, {rhv-full}, or {osp} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+
+This release has the following technical changes:
+
+.Technical change
+
+Technical change
+
+[id="new-features-and-enhancements-26_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
+.Cross-Provider Features
+
+Cross-provider features. link:https://issues.redhat.com/browse/MTV-802[(MTV-802)]
+
+.RH-SDL
```

Comment:
this is a development task that doesn't need to be documented in the release notes